cmd/podman/common.go

```diff
@@ -131,6 +131,10 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"cap-drop", []string{},
 		"Drop capabilities from the container",
 	)
+	createFlags.String(
+		"cgroup", "host",
```

Comment:
I think we should probably have a configurable default for this in libpod.conf, but we can add that later.

Reply:
(I want to be able to set a bool and default my containers to having a cgroup namespace)